frontend/src/types/API.ts

```diff
@@ -0,0 +1,8 @@
+export interface APIResponseOk {
```

Comment:
I'm not really happy with this, maybe we can just generate everything from the openAPI/swagger definition, that the backend offers (I was really for graphQL, but that ship had already sailed, when I started working on this)? Then we should have correct types and hopefully also correct error types.

Have been looking at this for e.g. https://github.com/YBogomolov/fetcher-ts